Tests/ConfigTests/ConfigurationUnitTests.cs

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using nanoFramework.TestFramework;
+
+namespace nanoFramework.Hardware.Stm32.UnitTests
+{
+    [TestClass]
+    public class ConfigurationTests
+    {
+        [TestMethod]
+        public void AddRevemoAdcChannels()
+        {
+            // comment if running on a real target
+            Assert.SkipTest();
```

Comment:
what you can use to make it nicer is this pattern (see https://github.com/nanoframework/System.IO.Ports/blob/2df511521ca2f1b8af36dd4d0746d2a5916d8050/Tests/UnitTestsSerialPort/SerialTests.cs#L23):
- Use the setup function
- create a try catch
- in the try/catch, place the creation of a channel
- if that works => you are on a real hardware, keep this test method
- if it crashes, you'll Asset.SkipTest() in the catch and the rest won't be executed at all

Reply:
Good suggestion. Done!